AES.Entities/Tables/ApplicantUser.cs

```diff
@@ -27,5 +27,11 @@ public class ApplicantUser
         public DateTime DOB { get; set; }
 
         public virtual UserInfo UserInfo { get; set; }
+
+        // Only one per applicant
+        public virtual Availability Availability { get; set; }
+
+        public DateTime? CallStartTime { get; set; }
```

Comment:
We should handle that on the code side, but those DateTime's should be required.